src/artemis/external_interaction/unit_tests/test_write_nexus.py

```diff
@@ -1,20 +1,80 @@
 import os
 from contextlib import contextmanager
 from pathlib import Path
+from typing import Literal
 from unittest.mock import call, patch
 
 import h5py
 import numpy as np
 import pytest
 from dodal.devices.fast_grid_scan import GridAxis, GridScanParams
+from scanspec.core import Path as ScanPath
+from scanspec.specs import Line
 
+from artemis.external_interaction.nexus.nexus_utils import create_goniometer_axes
 from artemis.external_interaction.nexus.write_nexus import (
     FGSNexusWriter,
-    create_parameters_for_first_file,
-    create_parameters_for_second_file,
+    create_parameters_for_first_gridscan_file,
+    create_parameters_for_second_gridscan_file,
+)
+from artemis.external_interaction.unit_tests.conftest import (
+    create_gridscan_goniometer_axes,
+    create_rotation_goniometer_axes,
 )
-from artemis.parameters.external_parameters import from_file as default_raw_params
 from artemis.parameters.plan_specific.fgs_internal_params import FGSInternalParameters
+from artemis.parameters.plan_specific.rotation_scan_internal_params import (
+    RotationInternalParameters,
+)
+
+
+def test_generic_create_gonio_creates_same_gonio_as_defined_with_individual_params(
```

Comment:
Should: This test is great to confirm what we've done in this issue but seems a bit pointless going forward given the old way will no longer exist, suggest we remove it.